ecs/users/fields/custom/rule.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: rule
+  title: Wazuh's User Rule
+  description: Rule assigned to a role.
+  reusable:
+    top_level: true
```

Comment:
I don't think a rule would be added at top level.

Reply:
Fixed on commit https://github.com/wazuh/wazuh-indexer-plugins/pull/303/commits/46c50892690e8669afab8fe1552f2b409a4af745